docs/install-docker-quickstart.md

```diff
@@ -1,264 +1,84 @@
-# Docker quickstart
+# Docker Quickstart
 
-To start developing with Nakama, you’ll first need to install it on your development machine. It’s straightforward and takes just a few minutes. In this guide we’re going to focus on installing your Nakama development instance using Docker.
+[Docker](https://www.docker.com/) is the quickest way to download and start developing with Nakama. By using Docker you are able to:
 
-!!! summary "Recommended Approach"
-    Docker is the quickest way to download and get started with Nakama for development purposes. For production settings, we recommend that you install Nakama as a [binary](install-binary.md) to ensure all system resources are available to Nakama.
+* Install to a pristine environment
+* Concurrently install and run the required [CockroachDB](https://www.cockroachlabs.com/) database
```

Comment:
I think we can drop "required" as Postgres is also supported.
Minor, what do you think of changing "concurrently" to "easily" as it speaks to how Docker simplifies complex tech which stands out more than doing something concurrently?